tests/expected/attr-invariant/check_invariant/check_invariant.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright Kani Contributors
```

Comment:
This test is subsumed by the previous one.

Reply:
This test has been converted into a more interesting one.